packages/styles/settings-tools/_u.mu-preview-grid.scss

```diff
@@ -0,0 +1,24 @@
+@if variable-exists('mozaic-env') and $mozaic-env == development {
+  $grid-line-color: rgba(0, 100, 100, 0.1);
+  $grid-subline-color: rgba(0, 100, 100, 0.03);
+  $grid-shadow-color: #ececec;
+  $grid-magic-unit: #{$magic-unit}rem;
+
+  .mu-preview-grid {
```

Comment:
`mdu-magic-unit-grid {}` for 'mozaic dev utility magic unit grid'
in a `packages/styles/dev-utils/_qdu.magic-unit-grid.scss`

Reply:
fixed